test/test_detection.py

```diff
@@ -35,16 +39,15 @@ def test_iscasa(self):
         self.assertTrue(iscasa(casatable))
         self.assertFalse(islofarhdf5(casatable))
         self.assertFalse(isfits(casatable))
-        pytest.xfail(reason="FIXME: don't know why this should fail")
         self.assertEqual(detect(casatable), None)
 
     @requires_data(hdf5file)
     def test_ishdf5(self):
         # TODO: disable this for now, since casacore can't parse LOFAR hdf5
-        #self.assertTrue(islofarhdf5(hdf5file))
+        # self.assertTrue(islofarhdf5(hdf5file))
         self.assertFalse(isfits(hdf5file))
         self.assertFalse(iscasa(hdf5file))
-        #self.assertEqual(detect(hdf5file), LofarHdf5Image)
+        # self.assertEqual(detect(hdf5file), LofarHdf5Image)
```

Comment:
Are these commented out lines still true?  Either way, I would probably remove them if this isn't something that's implemented.

Reply:
Yes, I tried uncommenting those two lines 47 and 50, but that led to errors, one of them being that `LofarHdf5Image` is missing.

Those two lines, that are commented out, can be removed.